### PR TITLE
added consistent <code> tags to properties that should be in code

### PR DESCRIPTION
**Pull request by aaronli39**
Certain css properties had inconsistent <code> tags. In addition, fixed minor grammar mistakes to make text more readable and clear.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/css-flexbox/use-the-flex-direction-property-to-make-a-row.english.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ forumTopicId: 301110
 
 ## Description
 <section id='description'>
-Adding <code>display: flex</code> to an element turns it into a flex container. This makes it possible to align any children of that element into rows or columns. You do this by adding the <code>flex-direction</code> property to the parent item and setting it to row or column. Creating a row will align the children horizontally, and creating a column will align the children vertically.
-Other options for <code>flex-direction</code> are row-reverse and column-reverse.
+Adding the property <code>display: flex</code> to an element turns the element into a flex container. This makes it possible to align any children of the element into rows or columns. You do this by adding the <code>flex-direction</code> property to the parent item and setting it to <code>row</code> or <code>column</column>. Creating a row will align the children horizontally, while creating a column will align the children vertically.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/01-responsive-web-design/css-flexbox/use-the-flex-direction-property-to-make-a-row.english.md]: `Adding the property <code>display: flex</code> to an element turns the element into a flex container.` creates a tautology.

Comment (thecodingaviator) [inline curriculum/challenges/english/01-responsive-web-design/css-flexbox/use-the-flex-direction-property-to-make-a-row.english.md]:
```suggestion
Adding the property <code>display: flex</code> to an element turns the element into a flex container. This makes it possible to align any children of the element into rows or columns. You do this by adding the <code>flex-direction</code> property to the parent item and setting it to <code>row</code> or <code>column</code>. Creating a row will align the children horizontally, while creating a column will align the children vertically.
```

Comment (thecodingaviator) [inline curriculum/challenges/english/01-responsive-web-design/css-flexbox/use-the-flex-direction-property-to-make-a-row.english.md]:
```suggestion
Adding the property <code>display: flex</code> to an element turns the element into a flex container. This makes it possible to align any children of the element into rows or columns. You do this by adding the <code>flex-direction</code> property to the parent item and setting it to <code>row</code> or <code>column</column>. Creating a row will align the children horizontally while creating a column will align the children vertically.
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by moT01** [comment]
how's this coming @aaronli39?

**Comment by moT01** [comment]
Hey @thecodingaviator, do you want to take another look at this? Since it doesn't look like the user will be fixing your suggestions, you can either make the changes yourself - approve the PR as-is - or I would be fine just closing this as well.

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**